Dialog/MainDialog.xaml.cs

```diff
@@ -185,17 +199,17 @@ private void CheckUnsafeFiles()
 
             foreach (Outlook.Attachment item in _mail.Attachments)
             {
-                foreach (string keyword in notsafe)
-                {
-                    if (item.FileName.Contains(keyword))
+                try {
```

Comment:
```suggestion
                try 
                {
```

It seems good to open this file with Visual Studio and exec Ctrl + K, Ctrl + D (auto formatting).

https://learn.microsoft.com/ja-jp/visualstudio/ide/default-keyboard-shortcuts-in-visual-studio?view=vs-2022#bkmk_test-explorer-context-specific-shortcuts

Reply:
Thanks, I've updated codes with formatting by VS2019.